### PR TITLE
Account Page restyle to v3

### PR DESCRIPTION
**Pull request by quetool**
**Context:**

Currently Web3ModalFlutter relies heavily on WalletConnectModalFlutter package. We want to change that and completely decouple them both.

Since Web3ModalFlutter is going to be restyled with v3 designs this is a first step towards that direction.

**Changes:**

- Account page restyle to v3
- New section "What is a Network"

_Note:_ The PR points to an intermediate branch called feat/v3restyle as I will be adding breaking changes and we don’t want to break anything for devs that may be using the package. Once everything is done we will merge the intermediate branch with master

**Final goal:**

- To have a completely independent Web3Modal package with it's own v3 design style

### CODE REVIEW

File: example/sign/lib/widgets/session_widget.dart [hunk 1]
```diff
@@ -81,65 +71,36 @@ class SessionWidgetState extends State<SessionWidget> {
...
-      _buildChainEventsTiles(
-        chainMetadata,
-      ),
-    );
+    children.addAll(_buildChainEventsTiles(chainMetadata));
 
     // final ChainMetadata
     return Container(
-      width: double.infinity,
+      // width: double.infinity,
```
Comment (0xAsimetriq) [inline example/sign/lib/widgets/session_widget.dart:90]: Needed comments?

**Review by 0xAsimetriq — APPROVED** [review]
Had a quick scan, I can see some commented out code is present and there is minor issue with compass asset, not sure if it was exported incorrectly or we have an issue in figma?

Screenshot 2023-09-19 at 13 17 00

File: example/sign/lib/widgets/session_widget.dart (hunk 1 above)
Comment (quetool) [reply example/sign/lib/widgets/session_widget.dart:90]: I'll go over the comments now but everything inside `example/` folder will be changed and mostly removed as that is the example app.
Regarding the svg asset, good catch! That one was exported prior my work, I'll re-export it.

Comment (quetool) [reply example/sign/lib/widgets/session_widget.dart]: Fixed. Compass asset looks better but still weird, not sure why but probably export issues...?

**Review by 0xAsimetriq — APPROVED** [review]

**Pull request merged**